<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QPushButton" name="next_button">
      <property name="text">
       <string>Weiter</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_instructions">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QStackedWidget" name="stacked_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="start_page">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="text_welcome">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Willkommen bei &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;kochbar.de&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Damit wir dir deine zukünftigen Lieblingsrezepte liefern können,&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;brauchen wir ein paar Informationen über dich...&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:10pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Viel Spaß beim Kochen und Guten Appetit!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_username">
          <property name="text">
           <string>Wähle einen Username:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="input_username"/>
        </item>
        <item>
         <widget class="QLabel" name="label_level">
          <property name="text">
           <string>Wähle deinen bevorzugten Schwierigkeitsgrad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_level">
          <item>
           <property name="text">
            <string>keine Präferenz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>leicht</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mittel</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>schwer</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_budget">
          <property name="text">
           <string>Wähle dein Budget:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="input_budget">
          <item>
           <property name="text">
            <string>keine Präferenz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kleines Budget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mittleres Budget</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>höheres Budget</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="category_page_0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="4" column="0">
         <widget class="QPushButton" name="dips_and_spreads_overlap">
          <property name="text">
           <string>Dips + Aufstriche</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="breakfast_and_brunch_overlap">
          <property name="text">
           <string>Frühstück + Brunch</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="drinks_overlap">
          <property name="text">
           <string>Getränke</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="desserts_overlap">
          <property name="text">
           <string>Desserts</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="appetizers_and_snacks_overlap">
          <property name="text">
           <string>Appetizers + Snacks</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="side_dish_overlap">
          <property name="text">
           <string>Beilagen</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="category_page_1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="2" column="0">
         <widget class="QPushButton" name="seafood_overlap">
          <property name="text">
           <string>Fisch + Meeresfrüchte</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="meat_and_poultry_overlap">
          <property name="text">
           <string>Fleisch + Geflügel</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="veggie_overlap">
          <property name="text">
           <string>Vegetarisches</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pasta_and_noodels_overlap">
          <property name="text">
           <string>Pasta + Nudelgerichte</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="italy_and_italian_style_overlap0pizza_overlap">
          <property name="text">
           <string>Italienisches + Pizza</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="asian_style_overlap0stir_fry_overlap">
          <property name="text">
           <string>Asiatisches + Wokgerichte</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="category_page_2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <widget class="QPushButton" name="healthy_overlap0low_calorie_overlap">
          <property name="text">
           <string>Gesundes</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="casseroles_overlap">
          <property name="text">
           <string>Aufläufe</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="salad_overlap">
          <property name="text">
           <string>Salate</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="soups_stews_and_chili_overlap">
          <property name="text">
           <string>Suppen, Eintopf, Chillis</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="roasts_overlap">
          <property name="text">
           <string>Braten</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="cookies_overlap0candy_overlap">
          <property name="text">
           <string>Süßigkeiten + Kekse</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="unwanted_page">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Hast du eine Unverträglichkeit?
(Dir werden keine Rezepte vorgeschlagen, die in die Kategorie &quot;enthält Laktose&quot; bzw. &quot;enthält Gluten&quot; fallen.)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="lactose_checkbox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Laktose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="gluten_checkbox">
          <property name="text">
           <string>Gluten</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="recommendations_page">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Neue Rezepte aus altbewährten Lieblingskategorien:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="old_view"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Probier was Neues, das dir bestimmt auch schmeckt:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="new_view"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_error">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="rating_combobox">
      <item>
       <property name="text">
        <string>1 Stern</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 Sterne</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 Sterne</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4 Sterne</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5 Sterne</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="search_field_prototype">
      <property name="text">
       <string>Suche...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
